server/src/actors/game_actor.rs

```diff
@@ -37,10 +41,15 @@ pub enum GameLoopCommand {
 }
 
 impl GameActor {
-    pub fn new(config: GameConfig, max_players: u32, time_limit_seconds: u32) -> GameActor {
+    pub fn new(config: GameConfig, max_players: u32, time_limit_seconds: u32, room_token: String) -> GameActor {
         let (msg_tx, msg_rx) = channel();
+        
+        let redis_uri = crate::APP_CONFIG.redis_uri.clone().unwrap_or("redis://127.0.0.1/".into());
+        let redis_actor = RedisActor::new(redis_uri);
+        let redis_actor_addr = redis_actor.start();
```

Comment:
we should make only one connection to redis and reuse in the games

Reply:
Noted, I've updated to initialize single connection to redis